rpi_nio_install.sh

```diff
@@ -16,21 +16,16 @@ sudo apt-get install python3 -y -q
 sudo apt-get install python3-pip -y -q
 sudo pip3 install -U pip
 
-echo
-echo INSTALLING NIO CLI
-echo ------------------
-sudo pip3 install nio-cli -U
-
 echo
 echo LOOKING FOR EXISTING NIO BINARY
 echo -------------------------------
-nio_location="$(which nio_run)"
+nio_location="$(which niod)"
 
 if [ $nio_location ]; then
   echo ... FOUND AT $nio_location
 else
   echo ... INSTALLING NIO FROM SPECIFIED BINARY
-  sudo pip3 install $binary #nio_lite-20171006-py3-none-any.whl
+  sudo pip3 install `find ~ -name "nio_lite-$binary\-py3-none-any.whl" | head -n 1`
```

Comment:
Search for the nio lite with user specified version. Grab the 1st result and install it